Unable to generate images on ARM64 platforms

```
$ sudo -E PATH=$PATH USE_DOCKER=true DISTRO=fedora ARCH=arm make rootfs
...
/osbuilder/rootfs.sh: line 208: /usr/go/bin/go: cannot execute binary file: Exec format error
INFO: [OK] Pull Agent source code
INFO: Build agent
/osbuilder/rootfs.sh: line 212: pushd: /home/devimc/go/src/github.com/kata-containers/agent: No such file or directory
Makefile:14: recipe for target 'rootfs' failed
make: *** [rootfs] Error 1
...
```

--- a/rootfs-builder/rootfs.sh
+++ b/rootfs-builder/rootfs.sh
@@ -92,9 +92,19 @@ check_function_exist() {
 generate_dockerfile() {
 	dir="$1"
 
+	case "$(arch)" in
+		"aarch64")
+			goarch=arm64
+			;;
+
+		*)
+			goarch=amd64
+			;;
+	esac
+
 	readonly install_go="
-ADD https://storage.googleapis.com/golang/go${GO_VERSION}.linux-amd64.tar.gz /tmp
-RUN tar -C /usr/ -xzf /tmp/go${GO_VERSION}.linux-amd64.tar.gz
+ADD https://storage.googleapis.com/golang/go${GO_VERSION}.linux-${goarch}.tar.gz /tmp
+RUN tar -C /usr/ -xzf /tmp/go${GO_VERSION}.linux-${goarch}.tar.gz
 ENV GOROOT=/usr/go
 ENV PATH=\$PATH:\$GOROOT/bin:\$GOPATH/bin
 "

